Composer/packages/client/src/App.tsx

```diff
@@ -67,17 +71,19 @@ export const App: React.FC = () => {
       <div css={main}>
         <nav css={sideBar(sideBarExpand)}>
           <div>
-            <IconButton
-              ariaLabel={sideBarExpand ? formatMessage('Collapse Nav') : formatMessage('Expand Nav')}
-              css={globalNav}
-              data-testid={'LeftNavButton'}
-              iconProps={{
-                iconName: 'GlobalNavButton',
-              }}
-              onClick={() => {
-                setSideBarExpand(!sideBarExpand);
-              }}
-            />
+            <TooltipHost content={globalNavButtonText} directionalHint={DirectionalHint.rightCenter}>
+              <IconButton
+                ariaLabel={globalNavButtonText}
+                css={globalNav}
+                data-testid={'LeftNavButton'}
+                iconProps={{
+                  iconName: 'GlobalNavButton',
+                }}
+                onClick={() => {
+                  setSideBarExpand(!sideBarExpand);
```

Comment:
```suggestion
                  setSideBarExpand((current) => !current);
```

When using the current state to get to a new state, use the updater function instead.

Reply:
Will do.